packages/crayons-extended/custom-objects/src/components/form-builder/assets/form-mapper.json

```diff
@@ -486,7 +501,8 @@
         "count": 2000,
         "message": "maximumLimits.maxChoiceOptionsWarning"
       },
-      "RELATIONSHIP": { "count": 5, "message": "maximumLimits.lookups" }
+      "RELATIONSHIP": { "count": 5, "message": "maximumLimits.lookups" },
+      "DEPENDENT_FIELD": { "count": 50, "message": "maximumLimits.fields" }
```

Comment:
Are you sure the max count of 50 is correct for the dependent fields? Please check this once

Reply:
We are allowing to create maximum fields allowed. So, I've set as 50 now.

Reply:
I understand the logic but allowing all 50 as dependent fields out of 50 is wrong. Please confirm the max limit with the PM on this and update. I will mark this as resolved for now